yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -404,6 +405,14 @@ export class Sequencer {
 
     const newGlobalVariables = proposalHeader.globalVariables;
 
+    // Kick off the process of collecting and validating proof quotes here so it runs alongside block building
+    const proofQuotePromise = this.createProofClaimForPreviousEpoch(newGlobalVariables.slotNumber.toBigInt()).catch(
+      e => {
+        this.log.debug(`Failed to create proof claim quote ${e}`);
```

Comment:
```suggestion
        this.log.warn(`Failed to create proof claim quote ${e}`);
```
We'll never find out if this fails otherwise